.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  format:
+    name: Check Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8.0'
+          distribution: 'adopt'
+      - run: |
+          ./dev/javafmt
+          if [[ $(git diff) != "" ]]
+          then
+              echo "code format error, please run the following commands:"
+              echo "   mvn com.coveo:fmt-maven-plugin:format"
```

Comment:
```suggestion
              echo "mvn com.coveo:fmt-maven-plugin:format"
```

Reply:
BTW, why not running the same command to format the code?

Reply:
The space is intended for indentation.

> BTW, why not running the same command to format the code?

Fixed